### DISCUSSION TITLE
Allow transient inputs configuration

### DISCUSSION THREAD

**Discussion by lajp** [YaLTeR/niri#791] [Ideas]
Related to: https://github.com/YaLTeR/niri/discussions/773

I would like to add a keybinding which I could use to toggle my touchpad on and off. Currently it is only possible (AFAIK) with configuration hot-reloading, which isn't really an option for me (I'm using [the Nix flake](https://github.com/sodiboo/niri-flake) to manage my configuration).

I'm also capable of contributing code if this is something you'd find useful to have.

**Comment by YaLTeR** [comment]
Could be useful I guess. niri msg input ...
